### ISSUE TITLE
AngularJS: spaces in <option> value attr with ng-repeat

### ISSUE THREAD

**Issue by pc3b3r** [kangax/html-minifier#345]
From https://github.com/yeoman/generator-angular/issues/1052

My controller has

``` javascript
$scope.opts = [
        "opt1",
        "opt2",
        "opt3",
        "opt4",
        "opt5"
    ];
$scope.opt = "opt1";
```

The view is like the following 

``` html
<select ng-model="opt">
    <option value="">...</option>
    <option ng-repeat="option in opts">{{option}}</option>
</select>
```

Using `grunt serve` there are no problems and the result is:

![opt](https://cloud.githubusercontent.com/assets/3038109/6917049/69fe43fe-d7a1-11e4-8470-08b4c115e24c.png)

When I run `grunt build` the result is

``` html
<select>
    <option value="">...</option>
    <option value="opt1 ">opt1 </option>
    <option value="opt2 ">opt2 </option>
    <option value="opt3 ">opt3 </option>
    <option value="opt4 ">opt4 </option>
    <option value="opt5 ">opt5 </option>
</select>
```

White spaces? Why?

**Comment by hoatle** [comment]
I'm facing the same problem.

**Comment by dxdc** [comment]
Here too. Same issue. Anyone found a resolution?

**Comment by pc3b3r** [comment]
I moved the data into a new obj with the property "value"

``` javascript
$scope.opts = [
        "opt1",
        "opt2",
        "opt3",
        "opt4",
        "opt5"
    ];
$scope.toOpts = [];
$scope.toOpts.push({"value" : "..."});
angular.forEach($scope.opts, function(value) {
    $scope.toOpts.push({"value" : value});
});
```

and in the view something like

``` html
<select ng-options="opt.value as opt.value for opt in toOpts"></select>
```

Hope it helps

**Comment by dxdc** [comment]
thanks @pc3b3r ... seems like a bit of a hack, but it works. I'll think about it :) ... could also get by with a simple perl find/replace after minification

**Comment by alexlamsl** [comment]
https://github.com/kangax/html-minifier/issues/361 could be a likely cause for this issue

**Comment by alexlamsl** [comment]
This should now be fixed in `1.2.0` on npm. If any of you still encounter the problem please reopen this issue.
